docs/components/concepts/testing.md

```diff
@@ -0,0 +1,31 @@
+---
+id: testing
+title: "Testing"
+description: "Test implementation of a single BPMN task."
+---
+
+You can run a single BPMN task directly from the modeler to test your implementation, without the overhead of executing an entire process. Get immediate feedback on your task logic, variable mappings, and configuration, right where you're modeling.
```

Comment:
Let's ensure that the concept is tangible, and contains a screenshot / capture of "testing in progress", in either Desktop or Web Modeler. Cf. [example](https://docs.camunda.io/docs/next/components/concepts/element-templates/).

Consider: Should we make clear it is a Desktop and Web Modeler feature?